scripts/setup_languagetool.sh

```
@@ -0,0 +1,32 @@
+
+echo "Setting up LanguageTool for CPSKDB project..."
+
+# Check if virtual environment exists
+if [ ! -d ".venv" ]; then
+    echo "Creating virtual environment..."
+    python3 -m venv .venv
+fi
+
+# Activate virtual environment and install LanguageTool
+echo "Installing LanguageTool..."
+source .venv/bin/activate
```

Shellcheck (0.10.0)

[info] Not following: .venv/bin/activate was not specified as input (see shellcheck -x).

(SC1091)